plugins/5.mediaKeys.ts

```diff
@@ -37,9 +37,11 @@ export default defineNuxtPlugin(() => {
         switch (state) {
           case MediaPlayerStatus.Paused:
             navigator.mediaSession.playbackState = "paused";
+            window.electronAPI.setThumbarBtns("paused");
```

Comment:
in the browser electronAPI shouldn't exist. So we need to check if that object exists.
Maybe window.electronAPI?. is enough?

Reply:
Yes, tested it. That's enough.